src/components/PaginatedTable.js

```diff
@@ -66,7 +66,8 @@ export default function PaginatedTable({columns, rows}) {
     : columns.slice(0, -1);
 
   return (
-    <Paper className="table">
+    // <Paper className={(isAdminFlow) ? "adminTable" : "employeeTable"}>
+    <Paper className="table" elevation={(isAdminFlow) ? 2 : 0}>
```

Comment:
This component should not know about existence of Admin.  You can simply pass `elevation: true or false`